Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:17.10
+
+ENV ZEPHYR_BASE=/zephyr
+ENV ZEPHYR_GCC_VARIANT=zephyr
+ENV ZEPHYR_SDK_INSTALL_DIR=/opt/zephyr-sdk
+
+# tools pre-requisites
+RUN apt-get update \
+    && apt-get install -yq --no-install-recommends git cmake ninja-build gperf \
+               ccache doxygen dfu-util device-tree-compiler \
+               python3-ply python3-pip python3-setuptools xz-utils file \
+               wget make vim-tiny \
```

Comment:
The Zephyr linux getting started guide does not include ``wget`` or ``vim-tiny`` in the required tools (but I see you using ``wget`` later in this file).